src/app/profile.service.ts

```diff
@@ -1,18 +1,22 @@
 import { Injectable } from '@angular/core';
 import {Profile} from "./Profile";
 import { Observable, of } from 'rxjs';
+import {HttpClient} from "@angular/common/http";
 
 @Injectable({
   providedIn: 'root'
 })
 export class ProfileService {
 
-  constructor() { }
+  private profileUrl = 'http://localhost:8081/practitioner/profile'; // temp. Inject this in task 549
```

Comment:
This will come from api-endpoint-service once feature/ARTS-549-inject-api-endpoint is merged